modules/031-linstor/templates/linstor-node/linstorsatelliteset.yaml

```diff
@@ -0,0 +1,62 @@
+# Source https://github.com/piraeusdatastore/piraeus-operator/blob/master/charts/piraeus/templates/operator-satelliteset.yaml
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: linstor-node
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "linstor-node" "workload-resource-policy.deckhouse.io" "every-node")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: DaemonSet
+    name: linstor-node
+  updatePolicy:
+    updateMode: "Auto"
+{{- end }}
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  annotations:
+    helm.sh/hook: post-upgrade, post-install
+    helm.sh/hook-delete-policy: before-hook-creation
+  name: linstor-node
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "linstor-node")) | nindent 2 }}
+spec:
+  minAvailable: 0
```

Comment:
You need to set minUnavailable here to avoid deleting all pods at one by the vpa controller.

Reply:
This is normal behavior, as LINSTOR does not support communication between different versions of controller and satellites, it's better to remove them all and deploy at once.

Reply:
So why do we need PDB then?

Reply:
That's the reasonable question, just removed this in f6c7f5d6ac70f6148442cb87d462c4f53c3cf127